service/files/defaults.yaml

```diff
@@ -7,13 +7,8 @@ configs:
     wsgi:
       workers: 4
 
-    sources:
-      - region: "hooli-west-1"
-        elastic_src: "http://127.0.0.1:9200/log-*/log"
-
-    backend:
-      elastic: "http://127.0.0.1:9200"
-      elastic_index: "ms_health_idx_1"
+    sources: []
+    elastic_index: "ms_health_idx_1"
```

Comment:
True, we don't use this anymore so sources can actually be empty, but we still need ['backend']['elastic'], see /health/config.py (also, can you please adjust it in the same PR?)